public-images/public-images-how-to/launch-with-libvirt.rst

```diff
@@ -0,0 +1,147 @@
+.. _launch-libvirt:
+
+Launch QCOW images using libvirt
+================================
+
+Amongst the virtualization tools available for Ubuntu, there is `libvirt`_. `libvirt`_ is not a hypervisor, but rather a set of tools and libraries to manage various other hypervisors, such as `KVM`_ and `QEMU`_.
+
+
+Although there are many more `applications using libvirt`_, this guide will focus on common ones. Those can be installed with of the following command:
+
+.. code:: bash
+
+   # graphical client
+   sudo apt-get install virt-manager
+
+
+The *recommended packages* for the aforementioned package installations will also install the `QEMU`_ libvirt driver, along with the command-line programs which will be used in this guide.
+
+
+Creating cloud-init user-data
+-----------------------------
+
+Since we are dealing with cloud images here, we will need a cloud-init user-data file to configure the VM instance. Refer to
+:ref:`use-local-cloud-init-ds`.
+
+
+Find and download an image
+--------------------------
+
+Ubuntu cloud images are hosted on https://cloud-images.ubuntu.com/
+
+QCOW2 images (``.img``) are suitable for use with `libvirt`_ and the `QEMU`_ driver.
+Refer to :doc:`../public-images-reference/artifacts` for a description the various image types.
+
+Once you have identified a suitable image, download it. For example, the following would download the current daily Ubuntu 24.04 (noble) image for amd64 machines:
+
+.. code:: bash
+
+   wget https://cloud-images.ubuntu.com/noble/current/noble-server-cloudimg-amd64.img
+
+
+
+Launching through console
```

Comment:
```suggestion
Launching through terminal
```
This suggestion is based on the fact that "console" is also used below with a different context. Choosing a different word in the title would help avoid confusion based on name clash.

Reply:
I like neither. "command-line" feels more accurate.